src/libs/GrowlNotification/GrowlNotificationContainer/index.native.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Animated} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../../styles/styles';
+
+const propTypes = {
+    children: PropTypes.node.isRequired,
+    translateY: PropTypes.instanceOf(Animated.Value).isRequired,
```

Comment:
and these

Reply:
also could probably be generalized since they are the same in both files.

Reply:
Sure, you are right, I just generalized them.